recipe/meta.yaml

```diff
@@ -49,12 +48,12 @@ requirements:
     - click-plugins >=1.0
     - cligj >=0.5
     - importlib-metadata  # [py<310]
-    - six
-    - setuptools
```

Comment:
Are we 100% sure that setuptools or pkg_resources are not used anymore at runtime?

Reply:
setuptools was removed in v1.9.6, see https://github.com/Toblerity/Fiona/blob/c4d64311b6403c7241165d67a3b9cdd9a09243ca/CHANGES.txt#L146

Reply:
And `six` in 1.9a1 https://github.com/Toblerity/Fiona/blob/c4d64311b6403c7241165d67a3b9cdd9a09243ca/CHANGES.txt#L331

Reply:
And `pkg_resources` in 1.10a1 https://github.com/Toblerity/Fiona/blob/c4d64311b6403c7241165d67a3b9cdd9a09243ca/CHANGES.txt#L136 with previous work in 1.9.4 https://github.com/Toblerity/Fiona/blob/c4d64311b6403c7241165d67a3b9cdd9a09243ca/CHANGES.txt#L186

You would think that it would trip over something in amongst its 1800 tests but as the very reason that this was delayed was because `fiona` tripped over what appeared to be a simple bug in `shapely` that `shapely`'s own gazillion tests didn't pick up on then... 🤷

Reply:
>You would think that it would trip over something in amongst its 1800 tests but as the very reason that this was delayed was because fiona tripped over what appeared to be a simple bug in shapely that shapely's own gazillion tests didn't pick up on then... 🤷

Yes, I've seen maybe dozens of cases where I got tripped up by a missing pkg_resources package even if all tests were passing.

Reply:
My analysis consists of going back to 1.9.5, the last time `pkg_resources` appears in any source code as some sort of evidence that it was used before and isn't used now by _this_ package.

If something down the dependency tree is expecting `pkg_resources` at runtime then that's another thing.

Reply:
`pkg_resources` and `setuptools` are not present in the source code at version `1.10.1`.